conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/NestedStructSuite.scala

```diff
@@ -27,48 +27,56 @@ import za.co.absa.enceladus.utils.testUtils.SparkTestBase
 class NestedStructSuite extends FunSuite with SparkTestBase with NestedStructsFixture {
 
   test("Test Dynamic Conformance does not hang on many mixed conformance rules") {
+    implicit val featureSwitches: FeatureSwitches = FeatureSwitches()
+      .setExperimentalMappingRuleEnabled(false)
+      .setCatalystWorkaroundEnabled(true)
+      .setControlFrameworkEnabled(false)
+
     val conformed = DynamicInterpreter.interpret(
       nestedStructsDS,
-      standardizedDf,
-      experimentalMappingRule = false,
-      isCatalystWorkaroundEnabled = true,
-      enableControlFramework = false
+      standardizedDf
     )
 
     assert(conformed.count() == 20)
   }
 
   test("Test Dynamic Conformance does not hang on many uppercase conformance rules") {
+    implicit val featureSwitches: FeatureSwitches = FeatureSwitches()
+      .setExperimentalMappingRuleEnabled(false)
+      .setCatalystWorkaroundEnabled(true)
+      .setControlFrameworkEnabled(false)
+
     val conformed = DynamicInterpreter.interpret(
       nestedStructsUpperDS,
-      standardizedDf,
-      experimentalMappingRule = false,
-      isCatalystWorkaroundEnabled = true,
-      enableControlFramework = false
+      standardizedDf
     )
 
     assert(conformed.count() == 20)
   }
 
   test("Test Dynamic Conformance does not hang on many negation conformance rules") {
+    implicit val featureSwitches: FeatureSwitches = FeatureSwitches()
+      .setExperimentalMappingRuleEnabled(false)
+      .setCatalystWorkaroundEnabled(true)
+      .setControlFrameworkEnabled(false)
+
     val conformed = DynamicInterpreter.interpret(
       nestedStructsNegationDS,
-      standardizedDf,
-      experimentalMappingRule = false,
-      isCatalystWorkaroundEnabled = true,
-      enableControlFramework = false
+      standardizedDf
```

Comment:
I've been going through the code bottom-up, I'll stop commenting the same thing now. Applies to all  of these. You get the picture :)

Reply:
Yup 😉 , thanks for the spot.